app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transfers/TransfersScreenWrapper.kt

```diff
@@ -132,20 +144,71 @@ private fun DetailPane(
     navigator: ThreePaneScaffoldNavigator<DestinationContent>,
     hasTransfer: () -> Boolean,
 ) {
-
     val destinationContent = navigator.safeCurrentContent()
 
     if (destinationContent == null) {
         NoSelectionEmptyState(hasTransfer())
+    } else if (destinationContent.folderUuid != null) {
```

Comment:
Can this be turned into a `== null` check? Also, would it be easier to read as a `when` expression?